### DISCUSSION TITLE
Can RSSHub send messages to platforms as well? Otherwise, are there any open source tools that can (that someone would recommend)?

### DISCUSSION THREAD

**Discussion by ItsCubeTime** [DIYgod/RSSHub#8891] [Q&A] [answered]
Can RSSHub send messages to platforms as well? Otherwise, are there any open source tools that can (that someone would recommend)?

Would like to distribute RSS feeds across various platforms (although it doesnt necessarily need to be from an RSS feed)

**Comment by TonyRL** [comment, marked as answer]
RSSHub won't send any messages. Some RSS readers may have notification options like email, push notification, etc.

If you want to send message to multiple platforms, take a look at [Gotify](https://gotify.net) or [Apprise](https://github.com/caronc/apprise).
